openwhisk/index.md

```diff
@@ -28,11 +28,52 @@ For more details about how {{site.data.keyword.openwhisk_short}} works, see [Abo
 ## Setting up the {{site.data.keyword.openwhisk_short}} CLI
 {: #openwhisk_start_configure_cli}
 
-You can use the {{site.data.keyword.openwhisk_short}} command line interface (CLI) to set up your namespace and authorization key. 
-Go to [Configure CLI](https://new-console.{DomainName}/openwhisk/cli){: new_window} and follow the instructions to install it. 
-Note that you must have Python 2.7 installed on your system to use the CLI.
+The OpenWhisk command line interface (CLI) requires Python 2.7.
 
-After {{site.data.keyword.openwhisk_short}} is set up with the CLI, you can begin using it from the command line or through REST APIs.
+- If you cloned the OpenWhisk respository, you will find the CLI in `openwhisk/bin/wsk`.
```

Comment:
Should this section revert back to the original? perhaps the entire index.md file for that matter...